WordPress/WordPressStatsWidgets/LockScreenWidgets/Models/LockScreenWidgetViewModelMapper.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+struct LockScreenWidgetViewModelMapper {
+    let data: HomeWidgetData
+
+    func getLockScreenSingleStatViewModel(title: String, dateRange: String) -> LockScreenSingleStatViewModel {
+        LockScreenSingleStatViewModel(
+            siteName: getSiteName(),
+            title: title,
+            value: getViews(),
+            dateRange: dateRange,
+            updatedTime: data.date
+        )
+    }
+
+    func getStatesURL() -> URL? {
+        if let todayData = data as? HomeWidgetTodayData {
+            return todayData.statsURL
+        } else {
+            return nil
+        }
+    }
+
+    func getSiteName() -> String {
+        data.siteName
+    }
+
+    func getViews() -> String {
+        if let todayData = data as? HomeWidgetTodayData {
```

Comment:
I considered to have `LockScreenStatsWidgetData`, the reason I didn't do it in this PR is that would need to create a standalone `TimeLineProvider`.

As we discussed, I will add the Provider and Entry because some logic is not necessary for the lock screen, so I think it's good to create `LockScreenStatsWidgetData` as your suggestions, but I'd like to defer it to the PR for creating lock screen Provider and Entry, let this PR focus on UI layer.

Reply:
Okay, thanks for explaining. I see you mentioned it in the TODO so we can leave it for another PR.